src/app/components/pages/project/project-welcome-page/project-welcome-page.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Not actually testing anything, this test should be easy, just test that the text you expect to be displayed is.